server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@Ganeshkm10 you shouldn't need to duplicate the file here, should be able to use the same file used by the char driver.